File: Question1/JS/Q1.js
function calculatePower(base, exp) {
    let answer = 1;

    for (let i = 0; i < exp; i++) {
        answer = answer * base;
    }
    return answer;
}

let base = 2
let exp = 3
let output = calculatePower(base, exp);

console.log("answer is ", output)

File: Question1/Java/Q1.java
import java.util.Scanner;

public class Q1 {

    //using recursion to calculate power
    public static int calculatePower(int base, int exp) {
        if (power != 0) {
            return (base * calculatePower(base, exp - 1));
        } else {
            return 1;
        }
    }

    public static void main(String[] args) {
        int base;
        int exp;
        int output;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Please input base: ");
        base = scanner.nextInt();
        System.out.print("Please input exponent: ");
        exp = scanner.nextInt();

        output = calculatePower(base,exp);
        System.out.println(base + " ^ "+ exp + " = " + output);
    }
}
